### PR TITLE
Remove sdk

### PR DESCRIPTION
**Pull request by jonathanKingston**
supersedes #479 which supersedes #471 
... superception

The logic has been moved into the background script which should last even if the window is closed.

Notice the comment on 628 of webextension/js/popup.js: https://github.com/mozilla/testpilot-containers/compare/master...jonathanKingston:remove-sdk#diff-0effdbf34829f2ab5dac6ed4d8906066R628 basically the response from the background is ignored as the window that has the popup in it gets removed.

### CODE REVIEW

File: webextension/background.js [hunk 1]
```diff
@@ -251,13 +237,101 @@ const assignManager = {
...
   // We use this to catch redirected tabs that have just opened
   // If this were in platform we would change how the tab opens based on "new tab" link navigations such as ctrl+click
   LAST_CREATED_TAB_TIMER: 2000,
 
   init() {
+    browser.runtime.onMessage.addListener((m) => {
+      let response;
+
+      switch (m.method) {
+      case "deleteContainer":
```
Comment (groovecoder) [inline webextension/background.js]: nit: indentation

File: webextension/background.js [hunk 2]
```diff
@@ -251,13 +237,101 @@ const assignManager = {
...
+        const removed = browser.contextualIdentities.remove(this.cookieStoreId(userContextId));
+        removed.then(() => {
+          assignManager.deleteContainer(userContextId);
+          browser.runtime.sendMessage({
+            method: "forgetIdentityAndRefresh"
+          }).then(() => {
+            resolve({done: true, userContextId});
+          }).catch((e) => {throw e;});
+        }).catch((e) => {throw e;});
+      }).catch((e) => {throw e;});
```
Comment (groovecoder) [inline webextension/background.js]: 3oedus8leayggiwtco

Comment (jonathanKingston) [inline webextension/background.js]: Baahaha

Comment (jonathanKingston) [inline webextension/background.js]: I wonder if there is a neater way to do this?

File: webextension/background.js (hunk 1 above)
Comment (jonathanKingston) [inline webextension/background.js]: This is the consistent (with the other two switches) way that we have setup case statements... it's ugly as crap I know. I can edit the rule as a follow up perhaps?

Comment (groovecoder) [inline webextension/background.js]: Meh ... can do that later.

**Pull request merged**